x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -121,7 +121,8 @@ interface FetchIndexReturn {
 
 export const useFetchIndex = (
   indexNames: string[],
-  onlyCheckIfIndicesExist: boolean = false
+  onlyCheckIfIndicesExist: boolean = false,
+  onlyAggregatable: boolean = false
```

Comment:
nit: at some point this might be better off as named arguments as it is tricker and more error problem to read and maintain API's with multitude of arguments such as:

```ts
useFetchIndex([], false, false);
```

vs.

```ts
useFetchIndex({ indexNames: [], onlyCheckIfIndicesExist: false, onlyAggregatable: false });
```

I flip back and forth between named and non-named but usually when I end up with two booleans in a row I end up just using named arguments.

It also gives you the opportunity to use `undefined` as a valid value to say that you aren't looking to filter against things that are or not aggregatable if you want to allow filtering of non-aggregatables or other items in the future vs. not filtering.

Although you could use something like this:

```ts
onlyAggregatable: boolean | undefined
```

rather than true or false if you want to distinguish between being able to filter against aggregatable, non-aggregatable or not filtering at all. Depends I guess if you want to expand the filtering abilities to be arbitrary filtering of things or not.

No changes needed here, just pointing out some thoughts.